ethernet-view/src/styles/styles.scss

```diff
@@ -5,12 +5,16 @@ $background-color: #dce3eb;
 $title-color: #45677d;
 $normal-text-color: #373c43;
 $alternate-text-color: #505868;
-$orange: hsl(29, 88%, 57%);
+$orange: rgb(242, 142, 49);
```

Comment:
to keep consistency with the rest of definitions on the file change to `#ee7623`